src/test/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStepTest.java

```diff
@@ -146,7 +146,7 @@
                         + "}", true));
                 WorkflowRun b = p.scheduleBuild2(0).waitForStart();
                 SemaphoreStep.waitForStart("basics/1", b);
-                story.j.assertLogContains(Functions.isWindows() ? "Masking only exact matches of %USERNAME% or %PASSWORD%" : "Masking only exact matches of $USERNAME or $PASSWORD", b);
+                story.j.assertLogContains(Functions.isWindows() ? "Masking supported pattern matches of %USERNAME% or %PASSWORD%" : "Masking supported pattern matches of $USERNAME or $PASSWORD", b);
```

Comment:
Feel free to introduce a package-private utility method for this message to be shared between `src/main/` and `src/test/`.